docs/resources/fivepunch-cops-and-robbers/index.md

```diff
@@ -0,0 +1,508 @@
+
+```cfg
+ensure fivepunch-cops-and-robbers
+```
+
+---
+
+## Configuration
+
+You can modify all configurations about minigame and matchmaking in file `config.lua`. But don't worry, you can safely use all default options.
+
+```lua
+LOCALE = {
+  LANGUAGE = 'en' -- Default language
+}
+
+GAME_CONFIG = {
+  MAX_ROUNDS = 8, -- Maximum rounds of each match
+  TIME_TO_ROUND_END_IN_MS = 1 * 60 * 1000, -- Time for the round to end giving victory to robbers
+
+  TIME_TO_DESTROY_GAME = 30 * 1000, -- Timeout to make a cleanup after match finished
+
+  DEFAULT_TIME_TO_HIDE_NOTIFICATIONS_IN_MS = 10 * 1000, -- Default time to hide notification
+  TIME_WAITED_TO_START_SPECTATING_IN_MS = 5 * 1000, -- Time after dead to start spectating teammate
+
+  TEAM_COLORS = {
+    COPS = "#39608b",
+    ROBBERS = "#82412d"
+  } -- Team related colors
+}
+
+MATCHMAKING_CONFIG = {
+  TIME_TO_START_ROUND_IN_MS = 15 * 1000, -- Time to first round start
+  TIMEOUT_TO_START_MATCH_IN_MS = 15 * 1000, -- Timeout after create a match to start first round
+
+  MAX_ITEMS_PER_PAGE = 20 -- Maximum rooms per page when request all rooms
+}
+
+BUCKET_CONFIG = { -- Configuration related to Fivem routing bucket
+  POPULATION_ENABLED = false,
+  LOCKDOWN_MODE = 'strict'
+}
+
+INPUT_CONFIG = { -- Configuraton related to Fivem key mapping
+  SCOREBOARD = {
+      KEY = "tab", -- Default key
+      MAPPER = "keyboard", -- Default mapper (optional)
+  },
+  SELECT_LONG_WEAPON = {
+      KEY = "1",
+      MAPPER = "keyboard",
+  },
+  SELECT_SHORT_WEAPON = {
+      KEY = "2",
+      MAPPER = "keyboard",
+  },
+  SELECT_MELLE_WEAPON = {
+      KEY = "3",
+      MAPPER = "keyboard",
+  },
+  SPECTATE_NEXT_PLAYER = {
+      KEY = "MOUSE_LEFT",
+      MAPPER = "MOUSE_BUTTON",
+  }
+}
+```
+
+---
+
+## Integrating
+
+By design, we don't bake framework support in our scripts. All of our creations are standalone, but they can be easily integrated with the framework of your preference.
+
+Here are some examples of how to integrate this script to some of the most common frameworks in the FiveM community.
+
+<Tabs>
+
+<TabItem value="standalone" label="Standalone">
+
+This resource is default standalone, you can easy download and install.
+
+</TabItem>
+
+<TabItem value="qb" label="QB">
+
+This resource is standalone, but to make the better compatibility in qb-core need some changes in others resource, some like remove default HUD, inventory, health system and others stuffs when playing.
+
+This compatibility will be written soon.
+
+</TabItem>
+
+<TabItem value="esx" label="ESX">
+
+This resource is standalone, but to make the better compatibility in esx need some changes in others resource like HUD, inventory, health system and others stuffs when playing.
+
+This compatibility will be written soon.
+
+</TabItem>
+
+</Tabs>
+
+---
+
+## How to add a new Map
+
+1. Create a json file inside `data/maps` folder, example `data/maps/shop_02.json`.
```

Comment:
1. Create a JSON file inside the `data/maps` folder, for example, `data/maps/shop_02.json`.